triggers/daml/Daml/Trigger/Internal.daml

```diff
@@ -56,8 +59,8 @@ addCommands : Map CommandId [Command] -> Commands -> Map CommandId [Command]
 addCommands m (Commands cid cmds) = Map.insert cid cmds m
 
 #ifdef DAML_GENMAP && DAML_GENERIC_COMPARISON
-adjust : forall k v. Ord k => k -> (Optional v -> Optional v) -> GMap.Map k v -> GMap.Map k v
-adjust k f m =
+alter : forall k v. Ord k => k -> (Optional v -> Optional v) -> GMap.Map k v -> GMap.Map k v
```

Comment:
In Haskell, the order of key and the modifier function is flipped. Not an issue here but once we move this to `daml-stdlib`,  we should probably try to match that unless we have a good reason not to do so.

Reply:
Might as well do it now. 4cba20c0dc